rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,170 @@
+
+# Detailed Design
+[design]: #detailed-design
+
+## Goals
+
+- Adopt the *Memorandum of Understanding on Equitable Moderation* to foster a shared understanding of equitable moderation practices that accommodate a community with diverse sociopolitical views.
+- Ensure transparent accountability of moderation practices to community members.
+- Ground community member accountability in publicly available general principles, providing a stable basis for behavioral expectations.
+- Make wide-ranging decisions regarding community member exclusion understandable.
+- Promote a diverse moderation team to balance diverse perspectives on controversial issues.
+- Establish an *Appeals Council* to hold moderation team members and procedures accountable and to allow for the revision of significant decisions if they fail to meet the community's reasonable standards.
+
+## Memorandum of Understanding on Equitable Moderation
+
+In response to the impression left by past events and with a fervent desire for improvement, we adopt the following principles and maxims as the *Memorandum of Understanding on Equitable Moderation*. These principles serve to clarify the overlapping consensus on reasonable expectations for community participation:
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
+5) We affirm that moderation actions must be fair and proportionate to the related issue.
+6) We ensure that concrete reasons for permanent bans are available for inspection by long-standing community members.
+7) We recognize that moderation actions should not be based solely on the volume of complaints.
+8) We commit to treating community members with respect and avoiding derogatory language when addressing concerns.
+9) We affirm our role as moderators is to facilitate a respectful community, not to punish members. We will use language that reflects mutual respect.
+10) We prefer inclusion to exclusion, resorting to permanent exclusion only in cases of extreme necessity. We are committed to exploring ways to prevent exclusion and to engaging constructively in efforts to reinstate excluded members.
```

Comment:
Participating in the Nix community is a privilege and not a right. The standard "only in cases of extreme necessity" is too high a bar.

Reply:
Yet to exclude someone is often bullying

Reply:
Moderation is *always* going to filter people out, *by design*. This is unconditional, because even a lack of moderation –as proposed here– will simply filter out a *different* set of people.

Saying "we don't want to exclude anyone" effectively means "we want different filters". But of course the former sounds a lot more nice

Reply:
Yes. Moderators need to filter out the spammers, trolls, and generally speaking bad actors. They need to exclude some people.

Reply:
I don't see how this is misaligned with what point 10 indicates. Shouldn't the moderation, **as a general rule**, try to exert their powers for inclusion rather than exclusion?

As an example, publicising my Nix flake a couple of times in Discourse (depending on the unit of time and how I do it) could make me a "spammer" in the eyes of some community member or moderator. Should a prospective moderator exclude me or perhaps should work with me so it doesn't happen again?

Of course the exclusion should remain a possibility. Perhaps I'm continuously opening sock puppet accounts to insult some people, if I'm caught I think a ban should be expected, but for the **majority** of cases in which moderation action should happen, I think it's a good thing that inclusion is the goal.

Reply:
> Moderation is _always_ going to filter people out, _by design_. This is unconditional, because even a lack of moderation –as proposed here– will simply filter out a _different_ set of people.
> 
> Saying "we don't want to exclude anyone" effectively means "we want different filters". But of course the former sounds a lot more nice

Someone voluntarily not participating is very different from barring someone form participating.

Reply:
> Excluding someone happens so we can protect the community. Framing that as bullying is a questionable mischaracterization of the purpose of moderation.

I did not say is always, not did I say it usually is in the case of Nix moderators.  Just that exclusion often is bullying general, and so there's definitely a risk of that within moderation.  Bullying is also often not intended as bullying.  That's why having good reasons for the exclusion is important

Reply:
> Someone voluntarily not participating is very different from barring someone form participating.

"voluntarily not participating" is some pretty words for "icking people out". If you have toxic people in your community, then whether others go "voluntarily" or not does not matter that much for that purpose. Either you filter out the toxic people or the ones annoyed by it get filtered out by the toxic people.

Reply:
In a way this might be what is happening and why this RFC is needed. People are being "icked out" by moderation action (check how some people in the Zulip for governance say they are disengaging from the process).

This RFC aims to increase accountability which in turn can prevent (some) toxic behaviors from appearing in the moderation team, where it can be difficult to root out.

I actually think we are talking about the same thing!

Reply:
> "voluntarily not participating" is some pretty words for "icking people out".

I don't agree.  A simple example, imagine you have a group of friends organizing a gathering- one friend(Person A) says "will Person B be there, if so I'm not going, they're annoying."  Is it exclusionary to allow Person B to come?  I don't think so.  It's different though if Person B is intentionally trying to upset Person A- but I don't believe most people want to hurt others.

> If you have toxic people in your community, then whether others go "voluntarily" or not does not matter that much for that purpose. Either you filter out the toxic people or the ones annoyed by it get filtered out by the toxic people.

If their annoyed by it, no I don't think that is justification for exclusion. Actually talking about bullying, that's the stereotypical reason people are bullied- being perceived as annoying.  If someone's actually being *harmful* moderators should first explain why it is harmful.  If they then prove to be intentionally harmful, that is when it is fair to exclude.

Reply:
So you think that –for example– being a careless dick in reviews would be okay? Because it's clearly not intentionally hurting others. And if people decide to avoid that person that's their choice.

And again, framing any exclusion as bullying is disingenuous. Yes both involve kind of getting rid of a person, and that's where the similarities end. Bullying is so much diff than politely showing someone the door.

Reply:
I didn't say being unintentionally hurtful is okay.  I said it's very different from being intentionally hurtful and should be treated differently.  

> And if people decide to avoid that person that's their choice.

Avoiding someone isn't exclusion, excluding someone is exclusion.  Threatening to leave a community if someone isn't excluded is calling for exclusion because you personally dislike someone.  Most online discussion forums have ways to block seeing some users, I'm not sure if Discourse has a way though as I can only see the option to mute.  That does seem like a valuable feature though if any person is so irritating to you.  

> And again, framing any exclusion as bullying is disingenuous.

It often(not always) is in my opinion.  You're free to disagree, but its not disingenuous because I'm being genuine.  

> Yes both involve kind of getting rid of a person, and that's where the similarities end. Bullying is so much diff than politely showing someone the door.

A community forum isn't someone barging into a private home.  If someone joins a local community group, then says something(not hurtful) that the leadership of the group disagrees with so they band together to kick them out- yeah that could be a form of bullying.

Reply:
> I didn't say being unintentionally hurtful is okay. I said it's very different from being intentionally hurtful and should be treated differently.

To add to this, some of the symptoms described by ASD and other disorders can make it difficult for people to interpret and predict the impact their words have on others.

Reply:
> > I didn't say being unintentionally hurtful is okay. I said it's very different from being intentionally hurtful and should be treated differently.
> 
> To add to this, some of the symptoms described by ASD and other disorders can make it difficult for people to interpret and predict the impact their words have on others.

Please don't try to use ASD and similar things do excuse rude behavior. I know many neurodivergent people that go out of their way to be kind to others, while there are also people that use their neurodivergency as a shield from skepticism.

Reply:
> Please don't try to use ASD and similar things do excuse rude behavior. 


> > > I didn't say being unintentionally hurtful is okay. I said it's very different from being intentionally hurtful and should be treated differently.
> > 
> > 
> > To add to this, some of the symptoms described by ASD and other disorders can make it difficult for people to interpret and predict the impact their words have on others.
> 
> Please don't try to use ASD and similar things do excuse rude behavior. I know many neurodivergent people that go out of their way to be kind to others, while there are also people that use their neurodivergency as a shield from skepticism.

I didn't and I'm not.  ASD has the S for a reason though.  And its important to recognize that the differences in people diagnosed with ASD is no less wide than the differences between neurotypicals.  What I'm saying is that something that might seem obviously rude to some people might not to others(ASD or not).  I only mentioned ASD to say that that could be something that exasperates that difference.

> out of their way to be kind to others

And I think this is also a big misunderstanding.  Saying someone may not be equipped to recognize how their words effects others doesn't mean they don't want to be kind.  Of course they do, most people do.  That doesn't mean that's how what they say is interpreted by others though.